docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -226,6 +226,27 @@ For more information regarding setup of the PostgresSQL connection, see `Postgre
 
      hba
 
+Setting up a MsSQL Database
+---------------------------
+
+You need to create a database and a database user that Airflow will use to access this database.
```

Comment:
Should we add warning that support for MSSQL is experimental?

Reply:
I think it's not - once we release 2.2 it's not going to be experimental